_docs/installation.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Docker Installation
+permalink: /docs/installation/
+layout: docs
+description: How to install Simpl using Docker
+---
+
+## Docker Installation
+
+You will need to have [Docker](https://www.docker.com/) installed.
+
+At the end of this tutorial, you will have successfully installed and played a single-player `Calc` game.
+
+In order to play a game, you will need to install and run `simpl-games-api`, a game model service, and a game UI.
+
+For every game, you will write the game model service, and the game UI. 
+
+For this tutorial, we will use [simpl-calc-model](https://github.com/simplworld/simpl-calc-model) and [simpl-calc-ui](https://github.com/simplworld/simpl-calc-ui)
+
+First, clone the [simple-games-api](https://github.com/simplworld/simpl-games-api) repository:
+
+```bash
+git clone https://github.com/simplworld/simpl-games-api
+```
+
+The `simpl-games-api` should always be installed first and will run the entire time.
+
+`cd` into the `simpl-games-api` repo and start Docker.
+
+```bash
+cd simpl-games-api
+docker-compose up
+```
+
+Open another terminal window. In that window, use a `make` script to log into the container, which is named `api`.
+
+```bash
+make shell
+```
+
+You will see in the window that you are now logged into the container.
+
+```bash
+Creating simpl-games-api_api_run ... done
+root@b2397c235f68:/code# 
+```
+
+Create a `superuser` and `staff user` by running `./manage.py simpl_setup` 
+
+```bash
+root@b2397c235f68:/code# ./manage.py simpl_setup
+```
+
+`superuser` is used to access the data in the Simpl API's Django admin. `staff user` is used by Simpl model services to connect to the Simpl API.
+
+Create a `superuser` when prompted. `superuser` can have any email address and password.
+
+```bash
+=== Setup Simpl API ===
+
+We found no superusers, please set one up below
+--- Creating a superuser ---
+Please enter email address for superuser: 
+```
+
+Create a `staff user` when prompted. `staff user` must use the email address `simpl@simpl.world` and password `simpl`, which correspond to the email address and password in the model service `settings.py` file.
+
+```bash
+We found no staff users, you will need one of these for your model service to connect to the Simpl API
+This is typically setup as a setting `SIMPL_GAMES_AUTH` from the environment variables `SIMPL_USER` and `SIMPL_PASS`
+
+--- Creating a staff user ---
+Please enter email address for staff user: 
+```
+
+`simpl-games-api` is now up and running. You can now exit and close that window.
+
```

Comment:
To be precise it was up and running when the docker-compose completed. What the user has just done is create Django users that are needed to run their game.

Reply:
I've corrected that.